modules/oe_content_news/oe_content_news.info.yml

```diff
@@ -10,6 +10,9 @@ dependencies:
   - oe_content:oe_content_reference_code_field
   - oe_media:oe_media
   - oe_media_avportal:oe_media_avportal
+  - oe_content_entity_contact:oe_content_entity_contact
```

Comment:
These modules have to be added to oe_content_news_post_update_00002()

Reply:
updated